source/src/molecular-unfolding/batch-images/sa-optimizer/process.py

```diff
@@ -78,16 +79,16 @@ def upload_result_files(execution_id, param_info, res_files: list, bucket):
 
 
 def sa_optimizer(qubo_model, model_file_info, param_info):
-    method = 'neal-sa'
+    method = sa_optimizer_method
     optimizer_param = {}
 
-    optimizer_param['shots'] =  1000
+    optimizer_param['shots'] =  10000
```

Comment:
any reason use the hardcode for `shots` count?

Reply:
shots hardcode is not hardcoded, this is default value, user can overwrite it from input.